src/components/layout/NavBar.jsx

```diff
@@ -89,51 +17,48 @@ class NavBar extends Component {
 
   render() {
     return (
-      <div style={{ width: '100%', backgroundColor: 'white', borderBottom: '1px solid #d1d1d1' }}>
-        <Header>
-          <NavLeft>
-            <NavLogo>
+      <div className="nav-bar">
+        <header className="nav-bar__header">
+          <nav className="nav-bar__nav nav-bar__nav--left">
```

Comment:
it doesn't look like `nav-bar__nav` is a class

Reply:
😆 Ah I am really bad at naming! how about `nav-bar__content`, `nav-bar__content--left`...

Reply:
the name is fine, I just didn't see the class anywhere in the .less file